elements/lisk-api-client/src/codec.ts

```diff
@@ -15,6 +15,7 @@
 
 import { codec, Schema } from '@liskhq/lisk-codec';
 import { utils } from '@liskhq/lisk-cryptography';
+import { standardEventDataSchema } from '@liskhq/lisk-chain';
```

Comment:
this needs to be redefined in the api-client to maintain node.js/ui lib separation